web/src/main/java/org/mskcc/cbio/portal/web/ApiController.java

```diff
@@ -45,6 +47,19 @@
     }
 
     @Transactional
+    @RequestMapping(value = "/mutation_count", method = {RequestMethod.GET})
+    public @ResponseBody List<DBAltCount> getMutations(@RequestParam(required = true) String type, @RequestParam(required = true) Boolean per_study, @RequestParam(required = true) List<String> ids, @RequestParam(required = true) List<String> genes, @RequestParam(required = true) List<Integer> starts, @RequestParam(required = true) List<Integer> ends, @RequestParam(required = false) List<String> echo) {
+        return service.getMutations(type, per_study, ids, genes, starts, ends, echo);
+        
+    }
+    
+    @Transactional
+    @RequestMapping(value = "/mutation_count", method = {RequestMethod.POST})
```

Comment:
Please try to follow the spec in the doc including the parameter names.

For example, can we support post of json? e.g.

```
[
  { 
    ‘data’: [
        {‘gene’: ‘geneA’,
          ‘startPos’: <startPosA>,
          ‘endPos’: <endPosA>,
          ‘myid’: ‘idA’},
          …
      ],
    ‘type’: ‘count’,
    ‘category’: ‘none’ or ‘study’,
    ‘echo’: [‘myid’]
  },
  {...},
]
```